elections/uk/lib.py

```diff
@@ -24,38 +26,43 @@ def shorten_post_label(post_label):
     'party_ec_id',
 ]
 
-def get_extra_csv_values(person, election):
-    theyworkforyou_url = ''
+def get_extra_csv_values(person, election, post):
+    gss_code = ''
```

Comment:
(on both comments), I agree that this is a change for a future PR. On the other one: the majority of candidates won't have TWFY URLs, so it might look a bit odd, but we can fix that another time.